src/main/java/no/finntech/maven/notifications/HipChat.java

```diff
@@ -75,18 +75,15 @@ protected void executeImpl() throws MojoExecutionException {
             throw new MojoExecutionException("\nhipchatMessage isn't defined.");
         }
 
-        if(!hipchatAnnouncement.exists()) {
-            throw new MojoExecutionException('\n' + hipchatAnnouncement.getAbsolutePath() + " doesn't exist.");
-        }
-
-
         final StringBuilder msg = new StringBuilder(hipchatMessage);
-        try {
-            msg.append("\n").append(new String(Files.readAllBytes(hipchatAnnouncement.toPath()), Charset.forName("UTF-8")));
-        } catch (IOException e) {
-            msg.append("failed to add release announcement. Due to: ").append(e.getMessage());
+        if(hipchatAnnouncement.exists()) {
+            try {
+                msg.append("\n").append(new String(Files.readAllBytes(hipchatAnnouncement.toPath()), Charset.forName("UTF-8")));
+            } catch (IOException e) {
+                msg.append("failed to add release announcement. Due to: ").append(e.getMessage());
+            }
```

Comment:
Actually, I just discovered that this patch won't do at all. The hipchat.announcement property is marked as required.

It should be changed to optional - and that means that the announcement file should be included if it's not null, and it exists. But if it is not null, and it doesn't exist, it should throw a MojoExecutionException.

Probably?